releasenotes-dca/notes/endpoint-support-for-prometheus-scraping-0b219cb1328dcc06.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Implementes endpoint support for Prometheus scraping in the services provider.
```

Comment:
let's not add a release note for now (all the previous PRs didn't have release notes as well), we'll add it when the whole feature is GA